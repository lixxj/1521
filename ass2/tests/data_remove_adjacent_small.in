a = malloc 10
b = malloc 10
c = malloc 10
d = malloc 10
free c
free b