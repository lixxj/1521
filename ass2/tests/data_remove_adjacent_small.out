+00000 (F, 5000) 
[a] +00008 
+00000 (A,   20) +00020 (F, 4980) 
[a] +00008 [b] +00028 
+00000 (A,   20) +00020 (A,   20) +00040 (F, 4960) 
[a] +00008 [b] +00028 [c] +00048 
+00000 (A,   20) +00020 (A,   20) +00040 (A,   20) +00060 (F, 4940) 
[a] +00008 [b] +00028 [c] +00048 [d] +00068 
+00000 (A,   20) +00020 (A,   20) +00040 (A,   20) +00060 (A,   20) +00080 (F, 4920)
[a] +00008 [b] +00028 [d] +00068 
+00000 (A,   20) +00020 (A,   20) +00040 (F,   20) +00060 (A,   20) +00080 (F, 4920)
[a] +00008 [d] +00068 
+00000 (A,   20) +00020 (F,   40) +00060 (A,   20) +00080 (F, 4920) 
+00000 (A,   20) +00020 (F,   40) +00060 (A,   20) +00080 (F, 4920) 
